is_core/auth_token/middleware.py

```diff
@@ -33,10 +33,12 @@ def process_response(self, request, response):
         # Save the session data and refresh the client cookie.
         # Skip session save for 500 responses, refs #3881.
         if response.status_code != 500 and hasattr(request, 'token') and request.token.is_active:
-            if not request.token.expiration:
```

Comment:
nene ta podmínka byla dobře
is-core má dvě možnosti
a) nastaví se expirace tokenu dle hodnoty AUTH_DEFAULT_TOKEN_AGE což je jedna hodina. A zároveň nenastavuje se platnost cookies což znamená že když zavřeš prohlížeč tak jsi odhlášený
b) při loginu zaklikneš tlačítko že chceš zůstat přihlášen což nastaví expiraci na cookies a to znamená že po zavření prohlížeče stále ještě budeš mít cookies a tedy i token. Platnost tokenu se pak řídí dle hodnoty AUTH_MAX_TOKEN_AGE která se rovná hodnotě AUTH_COOKIE_AGE což je dva týdny od jeho poslendího využití